src/components/macro/macro-action-editor/tab/macro-mouse/macro-mouse.component.ts

```diff
@@ -0,0 +1,104 @@
+import { EditableMacroAction } from '../../../../../config-serializer/config-items/macro-action/EditableMacroAction';
+import { MacroSubAction, macroActionType } from '../../../../../config-serializer/config-items/macro-action/MacroAction';
+import { Tab } from '../../../../popover/tab/tab';
+import { Component, Input, OnInit, ViewChild } from '@angular/core';
```

Comment:
Move imports from angular to the first line and add a new line after it, so it can be in a seperated group.